exasol_sagemaker_extension/lua/src/install_autopilot_prediction_udf.lua

```diff
@@ -0,0 +1,108 @@
+---
+-- @module install_autopilot_prediction_udf
+--
+-- This module generates and deploys Autopilot prediction udf
+--
+
+local M = {}
+
+local exaerror = require("exaerror")
+
+_G.global_env = {
+	pquery = pquery,
+	error = error
+}
+
+---
+-- Split the given string using the specified seperator into a list of strings
+--
+-- @param character		The separator to use when splitting the string.
+--
+-- @return  a list of strings
+--
+function string:split(seperator)
+  local splitted_result = {}
+  local i = 1
+  for str in string.gmatch(self, "[^"..seperator.."]+") do
+    splitted_result[i] = str
+    i = i + 1
+  end
+  return splitted_result
+end
+
+---
+-- Generate and deploy prediction udf
+--
+-- @param schema		       	The name of schema where the prediction udf gets created
+-- @param endpoint_name     	The name of endpoint is for which prediction UDF is created
+-- @param model_conn_name   	The name of connection holds information about mode
+-- @param input_params          Input parameters for the prediction udf
+-- @param output_params			Output for the prediction udf
+--
+function M.install_udf(schema, endpoint_name, model_conn_name, input_params, output_params)
+    local query_create  =
+	"CREATE OR REPLACE PYTHON3_SME SET SCRIPT " .. schema .. ".SME_ENDPOINT_PREDICTION_" .. endpoint_name .. "_UDF("
```

Comment:
check quoting of database identifiers

Reply:
the Prediction UDF should get a Prefix, because it is user defined.

Reply:
ahh yes we talked about this but I couldn't remember what to use as a prefix. we said not to use prefix, if i'm not wrong?

Reply:
yes, not prefix here